www/document/Document.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+jest.mock('next/document', () => ({
+    __esModule: true,
+    Html: ({ children }) => children,
+    Head: ({ children }) => <head>{ children }</head>, // eslint-disable-line react/prop-types
+    Main: () => <div />,
+    NextScript: () => <div />,
+    default: require('react').Component,
+}));
+
+import { Document } from './Document';
+
+beforeEach(() => {
```

Comment:
We can replace `beforeEach` by `beforeAll` because we don't need to inject `__NEXT_INTL_POLYFILL_URL__` variable before each test. In this test suite, this is not that important as we only have one test and `beforeEach` will run only once. But we should replace it so we can avoid this problem in the future.